src/components/item/grid.js

```diff
@@ -45,15 +45,13 @@ class ItemGrid extends ItemIterator {
   }
 
   get classes() {
-    return {
-      'item-grid': true,
+    return [super.classes, 'item-grid', {
       'drop-target': !this.props.isDisabled,
-      'over': this.props.isOver,
+      'dragging': this.state.isDragging,
```

Comment:
We can add `dragging` to iterator (probably `drop-target` as well?)

Reply:
done